main.go

```diff
@@ -0,0 +1,422 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+// DBInstanceClasses is used to store the available DB Instance Classes. The classes are specified with size order.
+var DBInstanceClasses = []string{
+	"db.t3.small",
+	"db.r5.large",
+	"db.r5.xlarge",
+	"db.r5.2xlarge",
+	"db.r5.4xlarge",
+	"db.r5.8xlarge",
+	"db.r5.12xlarge",
+	"db.r5.16xlarge",
+	"db.r5.24xlarge",
+}
+
+// SQSMessageBody is used to decode the SQS Message Body
+type SQSMessageBody struct {
+	Type             string `json:"type"`
+	MessageID        string `json:"messageId"`
+	TopicArn         string `json:"topicArn"`
+	Subject          string `json:"subject"`
+	Message          string `json:"message"`
+	Timestamp        string `json:"timestamp"`
+	SignatureVersion string `json:"signatureVersion"`
+	Signature        string `json:"signature"`
+	SigningCertURL   string `json:"signingCertURL"`
+	UnsubscribeURL   string `json:"unsubscribeURL"`
+}
+
+// Message is used to decode the SQS Message
+type Message struct {
+	AlarmName        string  `json:"alarmName"`
+	AlarmDescription string  `json:"alarmDescription"`
+	AWSAccountID     string  `json:"awsAccountId"`
+	NewStateValue    string  `json:"newStateValue"`
+	NewStateReason   string  `json:"newStateReason"`
+	StateChangeTime  string  `json:"stateChangeTime"`
+	Region           string  `json:"region"`
+	AlarmArn         string  `json:"alarmArn"`
+	OldStateValue    string  `json:"oldStateValue"`
+	Trigger          Trigger `json:"trigger"`
+}
```

Comment:
Are any of these objects something we could directly take from the AWS library? Not a big deal if not, but this seems like we are defining standard AWS objects? https://docs.aws.amazon.com/sdk-for-go/api/service/cloudwatch/#CompositeAlarm

Reply:
Hey @gabrieljackson Initially I tried to get these objects from AWS but couldn't find any, that is why I created them myself. I guess the reason is that we are trying to decode a Message that is being created by a CW alarm.